src/components/ApplicationContainer/index.tsx

```diff
@@ -25,7 +25,7 @@ const ApplicationContainer = ({ children, problemDetail }) => {
         skipNav={{ target: "main-content" }}
         navData={navConfig.current}
       />
-      <div className="nypl-library-card-app layout-container nypl-ds">
```

Comment:
Why no more `layout-container`?

Reply:
It wasn't being used. Maybe at one point it was getting styles for that class but that's no longer the case.